configx/provider_watch_test.go

```diff
@@ -133,10 +141,10 @@ func TestReload(t *testing.T) {
 		assert.Equal(t, "bar", p.String("foo"))
 
 		// but it is still watching the files
-		updateConfigFile(t, c, dir, name, "memory", "bar", "baz")
-		assert.Equal(t, "baz", p.String("bar"))
-
-		assertNoOpenFDs(t, dir, name)
+		updateConfigFile(t, c, dir, name, "memory", "foo", "bar")
+		assert.Equal(t, "bar", p.String("bar"))
+		assert.Equal(t, "foo", p.String("bar"))
```

Comment:
I don't think it makes sense to assert two different values for the same key here. Just revert the assertions here? If you run this test locally on a mac, it's also possible that it's a bit different from linux on the CI.

Reply:
derp accidental commit